bin/cat_consensus_sequences.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from Bio import SeqIO
+import click
+
+
+@click.command()
+@click.option("-s", "--sample-name", default="", help="Sample Name.")
+@click.option("-o", "--output-fasta", default="consensus.fasta", help="Consensus Fasta")
+@click.argument("consensus_sequences", nargs=-1)
+def write_consensus(sample_name, output_fasta, consensus_sequences):
+    sequence_list = []
+    for infile in consensus_sequences:
+        fasta_file = SeqIO.parse(open(infile), 'fasta')
+        for record in fasta_file:
+            seqid, sequence = record.id.strip(), record.seq
+            segment_number = seqid.split("_")[-2]
+            segment_name = seqid.split("_")[-1]
+            sequence_list.append([seqid, segment_number, segment_name, sequence])
+    sequence_list.sort(key=lambda tup: tup[1])
+    with open(output_fasta, 'w') as outfile:
+        for record in sequence_list:
+            outfile.write(f'>{sample_name + "_" + record[1] + " Segment " + record[2]}\n{record[3]}\n')
```

Comment:
Please use tuple value unpacking for readability:

```python
for _, seg_num, seg_name, seq in sequence_list:
    outfile.write(f'>{sample_name}_{seq_num} Segment {seq_name}\n{seq}\n')
```

Also, try to use only either f-strings or string concatentation, not both. It makes it very difficult to read and figure out what's going on.

And don't reuse variable names for very different things (`record` which is used for SeqRecord objects and then tuples).